src/services/downloadData.js

```diff
@@ -82,7 +82,9 @@ function getManifestDownload(type) {
 }
 export const fileManifestParticipantsOnly = getManifestDownload('participant');
 
-export const fileManifestParticipantsAndFamily = getManifestDownload('participant-family');
+export const fileManifestParticipantsAndFamily = async () => {
+  return getManifestDownload('participant-family');
+};
```

Comment:
unnecessary async without await. Also `getManifestDownload` is already a curried function, so I don't think we need this function wrapping